app/controllers/ArrivalMovementController.scala

```diff
@@ -42,9 +41,15 @@ import utils.CallOps._
 import utils.ResponseHelper
 import utils.Utils
 
+import java.time.OffsetDateTime
+import javax.inject.Inject
 import scala.concurrent.ExecutionContext
 import scala.xml.NodeSeq
-import controllers.actions.AnalyseMessageActionProvider
+
+@ImplementedBy(classOf[ArrivalMovementController])
+trait V1ArrivalMovementController {
+  def createArrivalNotification(): Action[NodeSeq]
+}
 
```

Comment:
make this a @Singleton

Reply:
Done the changes as suggested. Thanks Liandra